core/CronArchive/SegmentArchivingRequestUrlProvider.php

```diff
@@ -138,6 +158,15 @@ private function getCreatedTimeOfSegment($idSite, $segmentDefinition)
                 if ($createdTime->getTimestamp() < $earliestCreatedTime->getTimestamp()) {
                     $earliestCreatedTime = $createdTime;
                 }
+
+                if (!empty($segment['ts_last_edit'])) {
+                    $lastEditTime = Date::factory($segment['ts_last_edit']);
+                    if ($latestEditTime === null
+                        || $latestEditTime->getTimestamp() < $lastEditTime->getTimestamp()
```

Comment:
I think so, yes. Some may be for individual sites, some could be for all sites, and I don't see a unique constraint or any manual check in the API for duplicate segments. So looks possible to me.